preview/Dockerfile

```diff
@@ -1,28 +1,34 @@
-FROM buildpack-deps:trusty-scm
+FROM buildpack-deps:jessie-scm
 
 # Work around https://github.com/dotnet/cli/issues/1582 until Docker releases a
 # fix (https://github.com/docker/docker/issues/20818). This workaround allows
 # the container to be run with the default seccomp Docker settings by avoiding
 # the restart_syscall made by LTTng which causes a failed assertion.
 ENV LTTNG_UST_REGISTER_TIMEOUT 0
 
-# Install the LKG preview build of the .NET CLI. This is done by downloading
-# and installing the .deb file directly because there is a significant latency
-# between the publishing of a .NET CLI Debian package and its availability on
-# the package repository, so the simple "apt-get install dotnet" might not
-# pick up the latest build.
-#
-# Because the dpkg command cannot install dependencies, treat errors as
-# warnings and then use apt-get to install the dependencies.
-RUN apt-get update \
-    && wget https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/Latest/dotnet-host-ubuntu-x64.latest.deb \
-    && wget https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/Latest/dotnet-sharedframework-ubuntu-x64.latest.deb \
-    && wget https://dotnetcli.blob.core.windows.net/dotnet/beta/Installers/Latest/dotnet-sdk-ubuntu-x64.latest.deb \
-    && dpkg --force-depends -i dotnet-host-ubuntu-x64.latest.deb \
-    && dpkg --force-depends -i dotnet-sharedframework-ubuntu-x64.latest.deb \
-    && dpkg --force-depends -i dotnet-sdk-ubuntu-x64.latest.deb \
-    && apt-get -f -y --no-remove install \
-    && rm dotnet-host-ubuntu-x64.latest.deb \
-    && rm dotnet-sharedframework-ubuntu-x64.latest.deb \
-    && rm dotnet-sdk-ubuntu-x64.latest.deb \
+# Install .NET CLI dependencies
+RUN echo "deb [arch=amd64] http://llvm.org/apt/jessie/ llvm-toolchain-jessie-3.6 main" > /etc/apt/sources.list.d/llvm.list \
+    && wget -q -O - http://llvm.org/apt/llvm-snapshot.gpg.key|apt-key add - \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        clang-3.5 \
+        libc6 \
+        libcurl3 \
+        libgcc1 \
+        libicu52 \
+        liblldb-3.6 \
+        liblttng-ust0 \
+        libssl1.0.0 \
+        libstdc++6 \
+        libtinfo5 \
+        libunwind8 \
+        libuuid1 \
+        zlib1g \
     && rm -rf /var/lib/apt/lists/*
+
+# Install .NET CLI
```

Comment:
I changed the comment to `Install the LKG preview build of the .NET CLI.'  After this PR is merged, I am going to log a new issue to update the Dockerfile to use the Debian package once https://github.com/dotnet/cli/issues/2565 is fixed.  IMO it is not necessary to comment on this within the Dockerfile.